geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreTest.scala

```diff
@@ -48,12 +48,13 @@ class HBaseDataStoreTest extends Specification with LazyLogging {
     "work with points" in {
       val typeName = "testpoints"
 
+      System.setProperty("geomesa.hbase.remote.filtering", "true")
```

Comment:
should be able to use `SystemProperty("geomesa.hbase.remote.filtering").threadLocal.set("true")` (or something close to that) to avoid affecting other tests